src/main/kotlin/app/services/impl/HttpKeyService.kt

```diff
@@ -21,44 +23,65 @@ import java.net.URLEncoder
 @Service
 class HttpKeyService(private val httpClientProvider: HttpClientProvider) : KeyService {
 
+    companion object {
+        val logger: Logger = LoggerFactory.getLogger(HttpKeyService::class.toString())
+
+        // Will retry at 1s, 2s, 4s, 8s, 16s then give up (after a total of 31 secs)
+        const val maxAttempts = 5
```

Comment:
DW-2680 states a 10 maxRetry, should we stay consistent to that?

Reply:
No, I'll update the ticket - we looked at a reasonable total time.

Reply:
Ok!